src/app/zap-templates/templates/app/cluster-objects.zapt

```diff
@@ -129,11 +138,21 @@ struct DecodableType {
 public:
     static constexpr CommandId GetCommandId() { return Commands::{{asUpperCamelCase name}}::Id; }
     static constexpr ClusterId GetClusterId() { return Clusters::{{asUpperCamelCase parent.name}}::Id; }
+    static constexpr bool kIsFabricScoped = {{isFabricScoped}};
 
     {{#zcl_command_arguments}}
     {{zapTypeToDecodableClusterObjectType type}} {{asLowerCamelCase label}}{{> cluster_objects_field_init}};
     {{/zcl_command_arguments}}
+
+    {{~#if isFabricScoped~}}
+    {{#if (isClient source)}}
+    CHIP_ERROR Decode(TLV::TLVReader &reader, FabricIndex aAccessingFabricIndex);
```

Comment:
The relevant thing here, per Slack discussion, is whether the command has structs that are fabric-scoped as command fields, not whether the command itself is fabric-scoped, right?  Does anything in the spec actually require a command with a fabric-scoped struct field to itself be fabric-scoped?

Reply:
This is really more of a code requirement than a spec requirement - i.e. the command is the root, and so we need a way (at the root) to pass the fabric index to the underlying structs

What we could do (an alternative) would simply be to change all `Command::Decode` to always take in a `aAccessingFabricIndex`

I figured the advantage of the approach I've taken here is that it's more explicit, i.e. the caller will be aware that the command has data which is fabric scoped by having to pass in a fabric index

(We could imply from the sub-structs, but at that point we are effectively implicitly marking the command as fabric scoped, so seems reasonable to make it explicit)